2.9.0/miscategorized.md

```diff
@@ -5,5 +5,7 @@ Add any commits that were miscategorized for your domain below.
 Handle any commits that actually do belong to your domain and remove them from this list.
 
 ## Untopiced
+- [ROCm] Add FP8 rowwise support to _scaled_grouped_mm + Submodule update ([#159075](https://github.com/pytorch/pytorch/pull/159075))
```

Comment:
this one probably belongs with whatever release notes we normally put either AMD, or fp8 improvements?